contracts/contracts/core/BidderRegistryStorage.sol

```diff
@@ -36,15 +36,12 @@ abstract contract BidderRegistryStorage {
     /// Mapping from bidder addresses and window numbers to their funds per window
     mapping(address => mapping(uint256 => uint256)) public maxBidPerBlock;
 
-    /// @dev Mapping from bidder addresses to their locked amount based on bidID (commitmentDigest)
+    /// @dev Mapping from bidder addresses to their locked amount based on commitmentDigest
     mapping(bytes32 => IBidderRegistry.BidState) public bidPayment;
 
     /// @dev Amount assigned to bidders
     mapping(address => uint256) public providerAmount;
 
-    /// @dev Amount assigned to bidders
-    uint256 public blocksPerWindow;
```

Comment:
note: state change - relevant for upgrading

Reply:
N/a in main